sdk/formrecognizer/ai-form-recognizer/.vscode/launch.json

```diff
@@ -1,38 +0,0 @@
-{
-  // Use IntelliSense to learn about possible attributes.
-  // Hover to view descriptions of existing attributes.
-  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
-  "version": "0.2.0",
-  "configurations": [
-    {
-      "type": "node",
-      "request": "launch",
-      "name": "Current JS file",
-      "skipFiles": [
-        "<node_internals>/**"
-      ],
-      "program": "${file}",
```

Comment:
why removing launch on current file? I find it useful when debugging the samples.  My workflow:
1. build the library
2. change the import in a javascript sample to `import("../../")`
3. F5

Reply:
good point, I added those ones back.